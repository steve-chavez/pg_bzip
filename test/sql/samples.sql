create extension if not exists bzip;

select pg_read_file('./all_movies.csv') = convert_from(bzcat(pg_read_binary_file('./all_movies.csv.bz2')), 'utf8') as successful_decompression;

select pg_read_binary_file('./all_movies.csv.bz2') = bzip2(pg_read_binary_file('./all_movies.csv')) as successful_compression;
