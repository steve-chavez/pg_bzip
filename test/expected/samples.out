create extension if not exists bzip;
NOTICE:  extension "bzip" already exists, skipping
select pg_read_file('./all_movies.csv') = convert_from(bzcat(pg_read_binary_file('./all_movies.csv.bz2')), 'utf8') as successful_decompression;
 successful_decompression 
--------------------------
 t
(1 row)

select pg_read_binary_file('./all_movies.csv.bz2') = bzip2(pg_read_binary_file('./all_movies.csv')) as successful_compression;
 successful_compression 
------------------------
 t
(1 row)

